packages/react-instantsearch-dom-geo/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "react-instantsearch-dom-geo",
```

Comment:
It's not the final name, more a placeholder. Any inputs / ideas are welcome!

Reply:
Yes but since the next package will be called `react-instantsearch-dom` it makes more sense to have `react-instantsearch-dom-maps` WDYT?

Reply:
I would go for react-instantsearch-dom-geosearch (endname = widget name). About having dom before or after I have no preference, any examples of libraries named react-dom-xx vs react-xx-dom?

Reply:
I followed the same name as React Router `react-router-dom`, `react-router-native`. Most of the time the libs used for React DOM are just named `react-xxx`. But for React Native it's common to have `react-native-xxx`. Since we support both, make sense to follow the convention `react-{platform}-xxx`.